src/mphalport.h

```diff
@@ -9,6 +9,8 @@
 
 extern ringbuf_t stdin_ringbuf;
 
+#define mp_hal_ticks_cpu() (0)
```

Comment:
Need to understand this / investigate new time API.
We need to do the stubs for that too.

Reply:
Some discussion on the stubs PR too